### ISSUE TITLE
PIP install DI-engine failed.

### ISSUE THREAD

**Issue by linkerlin** [opendilab/DI-engine#594]
---------------------------------------- 62.8/62.8 kB 186.5 kB/s eta 0:00:00
ERROR: Cannot install di-engine==0.1.0, di-engine==0.1.1, di-engine==0.2.0, di-engine==0.2.1, di-engine==0.2.2, di-engine==0.2.3, di-engine==0.3.0, di-engine==0.3.1, di-engine==0.4.0, di-engine==0.4.1, di-engine==0.4.2, di-engine==0.4.3, di-engine==0.4.4, di-engine==0.4.5 and di-engine==0.4.6 because these package versions have conflicting dependencies.

The conflict is caused by:
    di-engine 0.4.6 depends on pettingzoo==1.12.0
    di-engine 0.4.5 depends on pettingzoo==1.12.0
    di-engine 0.4.4 depends on pettingzoo==1.12.0
    di-engine 0.4.3 depends on pettingzoo==1.12.0
    di-engine 0.4.2 depends on pettingzoo==1.12.0
    di-engine 0.4.1 depends on pettingzoo==1.12.0
    di-engine 0.4.0 depends on pettingzoo==1.12.0
    di-engine 0.3.1 depends on pettingzoo==1.12.0
    di-engine 0.3.0 depends on pettingzoo==1.12.0
    di-engine 0.2.3 depends on torch<=1.10.0 and >=1.1.0
    di-engine 0.2.2 depends on torch<=1.10.0 and >=1.1.0
    di-engine 0.2.1 depends on torch<=1.10.0 and >=1.1.0
    di-engine 0.2.0 depends on torch<=1.9.0 and >=1.3.1
    di-engine 0.1.1 depends on torch<=1.8.0 and >=1.3.1
    di-engine 0.1.0 depends on torch<=1.8.0 and >=1.3.1

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

**Comment by PaParaZz1** [comment]
Can you provide your environment information, such as: system version, Python version, PyTorch version and so on?

Maybe you can the output of the following code:
  ```python
  import torch, sys
  print(torch.__version__, sys.version, sys.platform)
  ```

**Comment by PaParaZz1** [comment]
I have fixed the problem of Pettingzoo in #597, you can checkout proper branch to install DI-engine from source code.

BTW, if you are in Windows system, please test and upload related result in your environment. PettingZoo doesn't officially support Windows platform.
